packages/atlas-meta-server/src/tags.ts

```diff
@@ -19,12 +20,12 @@ export type MetaTags = Record<string, string | number>
 
 const commonMetaTags = {
   'twitter:site': '@JoystreamDAO',
```

Comment:
If we're going through this exercise let's make this configurable via env as well